python/ray/tests/test_state_api.py

```diff
@@ -1599,6 +1615,11 @@ def verify():
     wait_for_condition(verify)
     print(list_actors())
 
+    if orig_external_dashboard_url:
```

Comment:
We don't need to clean up like this if we use `monkeypath` I believe. (but we will need do `ray.init` manually in the test rather than using the `shutdown_only` fixture).

Reply:
Yes,`ray.init` is already called in this test but can we still use `shutdown_only` to clean up the cluster after?